recipes/yacht/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "1.0" %}
+
+package:
+  name: yacht
+  version: {{ version }}
+
+source:
+  url: https://github.com/KoslickiLab/YACHT/releases/download/v1.0/yacht-{{ version }}-py_0.tar.bz2
```

Comment:
https://github.com/KoslickiLab/YACHT/releases/download/`v1.0`/yacht-{{ version }}-py_0.tar.bz
Can `v.1.0` be replaced by {{ version }} too ?